monitor/monitoring-dam-generator-core/resources/currentAtosAdpFromOptimizer.yml~

```diff
@@ -0,0 +1,119 @@
+tosca_definitions_version: tosca_simple_yaml_1_0_0_wd03
```

Comment:
Probably this file is a backup file of your editor. May we remove it from the PR?